packages/platform-sdk-btc-indexer/src/logger.ts

```diff
@@ -0,0 +1,12 @@
+export default class Logger {
```

Comment:
Do we have any other logger better than this humble one?

Reply:
That should work for now. I'll make one tomorrow that we can share across packages with no issues.